ui/src/components/pipelines/pipeline_designer/context_bar/transformation/Config.js

```diff
@@ -80,6 +95,12 @@ class Config extends Component {
           </div>
         </div>
         <div className="form-group mb-0 py-4 datacater-context-bar-function-config">
+          {!transformExpectsDataType && (
+            <div className="alert alert-warning">
+              The transform <i>{transform.key}</i> does not support the input
+              type <i>{attributeDataType}</i>.
```

Comment:
It is just providing hints based on label `input-types` in the `spec.yml` of the transform/filter ([example](https://github.com/DataCater/datacater/blob/main/transforms/capitalize/spec.yml#L11)) but not taking any hard decisions like preventing applying a specific transform to a field that does not feature a matching data type.

Reply:
Good question! I feel that we should keep these warnings as hints in the UI and not write them to the logs of the pipeline deployment for now.

It might be interesting to make these warnings part of the pipeline logs if we would introduce some kind of debug mode to pipeline deployments at some point. Thoughts?

Reply:
Concluding, that we should have a conversation around debug mode of pipelines. The arguments above make sense, hence resolving this conversation.